docs/docs/guide-tools/unit-yaml-schema.md

```diff
@@ -16,27 +16,63 @@ The schema for a Basis Unit defines several key properties:
-  - `multi`: Handlers run in parallel by default.
 
 - **handlers**: This section describes the message delivery logic, or handlers, that execute within the Unit. Each handler is defined by its synchronization type, which determines how and when the handler is triggered. The handler may require inputs to be satisfied, and it produces a set of outputs.
   
   - **sync**: The synchronization method, handled by a Syncrhonizer, for the handler. It can be defined as:
     - `all`: The handler is triggered when all inputs are satisfied.
     - `equal`: The handler is triggered when inputs match specified conditions.
-    - `rate`: The handler, referred to as a RateSubscriber, triggers at regular intervals specified by the `rate` field. In this case, no external inputs are required for the handler to be activated; it operates purely based on the defined time interval.
     - `approximate`: Optional inputs may be provided, and the handler will use approximations if necessary.  
 
-  - **buffer_size**: Specifies the maximum number of messages to buffer for each input. Once the buffer reaches the defined `buffer_size`, additional messages will be dropped.
+  - **rate**: When present, a RateSubscriber is applied to the synchronizeer. It triggers at regular intervals specified by the `rate` field.
 
+  - **buffer_size**: Specifies the maximum number of messages to buffer for each input. Once the buffer reaches the defined `buffer_size`, additional messages will be dropped.
  
   - **inputs**: A set of input topics that the handler waits for before execution. Each input is described by several properties, such as a synchronization field (`sync_field`), caching options, and whether the input allows or denies certain transport methods. Inputs are configured with a defined message type, and options such as accumulated data and quality of service (QoS) can be specified.
   
   - **outputs**: The outputs generated by the handler, which are typically messages published to other components. Similar to inputs, outputs are described by message types, transport methods, and other properties such as QoS settings.
 
+### Synchronization Field
+
+In each input, the `sync_field` property specifies how the field is accessed for synchronization. Depending on how the message data is structured, the `sync_field` can be set in various ways:
+
+- **Direct Field Access**: When a field can be accessed directly from the message, you can specify the field name. For example:
+
+  ```yaml
+  inputs:
+    /camera: 
+      type: rosmsg:sensor_msgs::Image
+      sync_field: ::width
+  ```
+
+  This corresponds to the code `msg->width`, directly accessing the `width` field in the `sensor_msgs::Image` message.
+
+- **Method Field**: If a method is used to retrieve the field, you can specify the method in the `sync_field`. For example:
+
+  ```yaml
+  inputs:
+    /lidar: 
+      type: protobuf:LidarMessage 
+      sync_field: ::NumBeams()
```

Comment:
You don't actually need or want () here, it won't compile as is, currently

Reply:
Removed